mkdocs.yml

```diff
@@ -4,12 +4,35 @@ site_author: Amazon Web Services
 repo_url: https://github.com/aws-powertools/powertools-lambda-typescript
 edit_uri: edit/main/docs
 site_url: https://docs.powertools.aws.dev/lambda/typescript
+watch: [
+  docs,
+  packages/batch/src,
```

Comment:
I'm adding each directory one by one because I don't want to add too many files to the watch list. We only care to reload the docs when the source changes, but we want to exclude tests, build files, etc.

I tried to use globs (i.e. `packages/*/src`) or find a way too exclude, but couldn't make it work.